hedera-node/hedera-app/src/test/java/com/hedera/node/app/throttle/HapiThrottlingTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.throttle;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import com.hedera.hapi.node.base.HederaFunctionality;
+import com.hedera.hapi.node.transaction.Query;
+import com.hedera.node.app.state.HederaState;
+import com.hedera.node.app.workflows.TransactionInfo;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class HapiThrottlingTest {
+
+    @Mock
+    private GeneralThrottleAccumulator generalThrottleAccumulator;
+
+    @Mock
+    private TransactionInfo transactionInfo;
+
+    HapiThrottling subject;
+
+    @BeforeEach
+    void setUp() {
+        subject = new HapiThrottling(generalThrottleAccumulator);
+    }
+
+    @Test
+    void verifyShouldThrottleIsCalled() {
+        // given
+        final var state = mock(HederaState.class);
+
+        // when
+        subject.shouldThrottle(transactionInfo, state);
+
+        // then
+        verify(generalThrottleAccumulator, times(1)).shouldThrottle(eq(transactionInfo), any(), eq(state));
```

Comment:
I believe `times(1)` is the default and unnecessary here.

Reply:
Yes, the default is `times(1)` but writing it explicitly seems more clear to me. Looking at the code we have a fair amount of asserts that are written that way. What do you think?